Client/Frontend/Browser/TabManager.swift

```diff
@@ -453,14 +453,40 @@ class TabManager: NSObject {
                     for delegate in self.delegates {
                         delegate.get()?.tabManagerDidAddTabs(self)
                     }
+                } else {
+                    self.showFocusPromoToast()
                 }
                 self.eraseUndoCache()
             })
         }
 
         delegates.forEach { $0.get()?.tabManagerDidRemoveAllTabs(self, toast: toast) }
     }
-    
+
+    func showFocusPromoToast() {
```

Comment:
Does this need a check for *iPad*? We do not have a tab manager there, and no empty state. Or is the TabManager simply not used on iPad at all?